### PR TITLE
updated mapboxSdkServices library: to DirectionRoute added `durationT…

### PR DESCRIPTION
**Pull request by RingerJK**
…ypical`

## Description

Delay information of step-by-step navigation. To `DirectionRoute` added `durationTypical` field

- [ ] I have added any issue links
- [ ] I have added all related labels (`bug`, `feature`, `new API(s)`, `SEMVER`, etc.)
- [ ] I have added the appropriate milestone and project boards

### Goal

Please describe the PR goals. Just the stuff needed to implement the fix / feature and a simple rationale. It could contain many check points if needed

### Implementation

Please include all the relevant things implemented and also rationale, clarifications / disclaimers etc. related to the approach used. It could be as self code companion comments

## Screenshots or Gifs

Please include all the media files to give some context about what's being implemented or fixed. It's not mandatory to upload screenshots or gifs, but for most of the cases it becomes really handy to get into the scope of the feature / bug being fixed and also it's REALLY useful for UI related PRs screenshot gif

## Testing

Please describe the manual tests that you ran to verify your changes

- [ ] I have tested locally (including `SNAPSHOT` upstream dependencies if needed) through testapp/demo app and run all activities to avoid regressions
- [ ] I have tested via a test drive, or a simulation/mock location app
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have updated the `CHANGELOG` including this PR
- [ ] We might need to update / push `api/current.txt` files after running `$> make core-update-api` (Core) / `$> make ui-update-api` (UI) if there are changes / errors we're 🆗 with (e.g. `AddedMethod` changes are marked as errors but don't break SemVer) 🚀 If there are SemVer breaking changes add the `SEMVER` label. See [Metalava](https://github.com/mapbox/mapbox-navigation-android/blob/master/docs/metalava.md) docs

### CODE REVIEW

**Comment by RingerJK** [comment]
@langsmith feel free to edit/add your changes for UI

File: libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/TelemetryLocationAndProgressDispatcher.kt [hunk 1]
```diff
@@ -33,7 +33,12 @@ internal class TelemetryLocationAndProgressDispatcher(scope: CoroutineScope) :
         AtomicReference(
             RouteProgressWithTimestamp(
                 0,
-                RouteProgress.Builder(DirectionsRoute.builder().build()).build()
+                RouteProgress.Builder(
+                    DirectionsRoute.builder()
```
Comment (RingerJK) [inline libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/TelemetryLocationAndProgressDispatcher.kt:37]: `distance` and `duration` [mustn't be null ](https://github.com/mapbox/mapbox-java/blob/aa126fcad9e7dba2f5b7f74c5a46291463d9b85e/services-directions-models/src/main/java/com/mapbox/api/directions/v5/models/DirectionsRoute.java#L50-L60)

File: build.gradle [hunk 2]
```diff
@@ -88,7 +88,7 @@ allprojects {
     }*/
     maven { url 'https://mapbox.bintray.com/mapbox' }
     // uncomment if snapshots access is needed
-    // maven { url 'https://oss.jfrog.org/artifactory/oss-snapshot-local/' }
+     maven { url 'https://oss.jfrog.org/artifactory/oss-snapshot-local/' }
```
Comment (RingerJK) [inline build.gradle:91]: be committed again, when mapbox-java is updated

Comment (LukasPaczos) [reply build.gradle:91]: Could we avoid merging snapshot repo references to the main branch? It's bitten us in the past as it slipped into a release. I'd rather have a snapshot tied to a feature PR that needs it and then bump the version of mapbox-java before merging. Alternatively, we can release a `-beta` of mapbox-java.

Comment (RingerJK) [reply build.gradle:91]: @LukasPaczos we aren't gonna merge PR with current changes, this change is here to show how it work and unblock @langsmith UI side work. We definitely gonna make a new mapbox-java release if everything is 🆗

**Comment by RingerJK** [comment]
captured from Slack: 
> `@Yury Kanetski`, heads up about `@lukaspaczos` ’ https://github.com/mapbox/mapbox-navigation-android/pull/3518#discussion_r481045682 comment about your delay info work. I’m working on the nav UI side of things but haven’t pushed my work to :github: yet. However, I did git merge kyv-route-delay-arriving into my branch so that I’d get access to your work, the Java SDK SNAPSHOT, etc.  `@lukaspaczos` suggestion is that we close your pr and just have my pr use a stable release when it eventually lands to our main Nav SDK branch.

**cc** @LukasPaczos @langsmith

**Pull request closed without merging**